go.mod

```diff
@@ -91,7 +91,7 @@ require (
 	github.com/pierrec/lz4/v4 v4.1.3 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.9.0
-	github.com/prometheus/client_model v0.2.0
+	github.com/prometheus/client_model v0.2.0 // indirect
```

Comment:
We can revert this. This is an unfortunate problem of our codebase.

Reply:
sure